tutorials-and-examples/inference-servers/jetstream/maxtext/single-host-inference/README.md

```diff
@@ -147,7 +149,55 @@ Deploy the manifest file for the Maxengine server and HTTP server:
 kubectl apply -f deployment.yaml
 ```
 
-## Verify the deployment
+### Deploy via Terraform
+
+Navigate to the `./terraform` directory and do the standard [`terraform init`](https://developer.hashicorp.com/terraform/cli/commands/init). The deployment requires some inputs, an example `sample-terraform.tfvars` is provided as a starting point, run `cp sample-terraform.tfvars terraform.tfvars` and modify the resulting `terraform.tfvars` as needed. Finally run `terraform apply` to apply these resources to your cluster.
+
+#### (optional) Enable Horizontal Pod Autoscaling via Terraform
```

Comment:
as discussed, using Terraform is good, but not all users would want to go with this approach. Please provide an approach to configure HPA with k8s yamls as another option

Reply:
Sure! I plan on immediately following up with the kubectl portion, separating them into two PRs since this one is already at 800+ lines.